app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -38,13 +37,22 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({
   async function handleSignupSubmit(rawUserData: UserDataValidation) {
     setIsLoading(true);
 
-    const sanitizedUserData = userDataSanitizationSchema.parse(rawUserData);
+    const { email, name, interests, source } = rawUserData;
+
+    const stringifiedInterests = JSON.stringify(interests);
```

Comment:
The `interests` input is already a string, so probably no need to stringify it.

Reply:
Good catch! I did that initially because we were using sanitization before storing in localStorage. I totally forgot to change it.